certifiers/errors/errors.go

```diff
@@ -0,0 +1,86 @@
+package errors
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+)
+
+var (
+	errValidatorsChanged = fmt.Errorf("Validators differ between header and certifier")
```

Comment:
the reason you are not exposing this because you want a user to use `IsXXXError` methods?

Reply:
Yes. You cannot compare, as we wrap with a stack trace. You need `errors.Cause(err) == certerr.errValidatorsChanged` and that is not so nice to expose.

Reply:
got it. cool